configs/caas/cromwell.conf.ctmpl

```diff
@@ -374,6 +374,11 @@ backend {
           }
 	  http {}
         }
+
+        memory-retry {
```

Comment:
Looks like this needs to go under the PAPIv2 backend, not JES according to the config docs

Reply:
Woops, good eye! I checked it in the online version, but then misplaced in the local checkout :)